apps/server/src/notifications/commands/mark-notification-as-read.command.ts

```diff
@@ -1,22 +1,30 @@
 import { injectableClass } from 'ditox';
+import { eq } from 'drizzle-orm';
 
 import { CommandHandler } from '../../infrastructure/cqs/command-handler';
-import { NotificationRepository } from '../../persistence/repositories/notification/notification.repository';
+import { DatePort } from '../../infrastructure/date/date.port';
+import { Database } from '../../persistence/database';
+import * as schema from '../../persistence/schema';
 import { TOKENS } from '../../tokens';
-import { NotificationNotFound, MemberIsNotNotificationRecipient } from '../notification-errors';
+import { MemberIsNotNotificationRecipient, NotificationNotFound } from '../notification-errors';
 
 export type MarkNotificationAsReadCommand = {
   notificationId: string;
   memberId: string;
 };
 
 export class MarkNotificationAsRead implements CommandHandler<MarkNotificationAsReadCommand> {
-  static inject = injectableClass(this, TOKENS.notificationRepository);
+  static inject = injectableClass(this, TOKENS.date, TOKENS.database);
 
-  constructor(private readonly notificationRepository: NotificationRepository) {}
+  constructor(
+    private readonly dateAdapter: DatePort,
+    private readonly database: Database,
+  ) {}
 
   async handle({ memberId, notificationId }: MarkNotificationAsReadCommand): Promise<void> {
-    const notification = await this.notificationRepository.getNotification(notificationId);
+    const notification = await this.database.db.query.notifications2.findFirst({
```

Comment:
**question**

are we keeping `notification2`?

Reply:
Nop, I need to merge all migrations into a single one.

Reply:
a70000e